modules/apps/29-fee/ibc_middleware.go

```diff
@@ -261,10 +258,9 @@ func (im IBCMiddleware) OnTimeoutPacket(
 	packetID := channeltypes.NewPacketId(packet.SourcePort, packet.SourceChannel, packet.Sequence)
 	feesInEscrow, found := im.keeper.GetFeesInEscrow(ctx, packetID)
 	if found {
-		im.keeper.DistributePacketFeesOnTimeout(ctx, relayer, feesInEscrow.PacketFees)
-
-		// removes the fee from the store as fee is now paid
-		im.keeper.DeleteFeesInEscrow(ctx, packetID)
+		if err := im.keeper.DistributePacketFeesOnTimeout(ctx, relayer, feesInEscrow.PacketFees, packetID); err != nil {
```

Comment:
we should adjust the API of DistributePacketFeesOnTimeout to also not return an error